### PR TITLE
Disable PlayerComponentsTest

### PR DESCRIPTION
**Pull request by zhongqiliang**
b/323250981

### CODE REVIEW

File: starboard/android/shared/test_filters.py
Comment (oxve) [inline starboard/android/shared/test_filters.py:1]: I think this looks good but I would like someone closer to the code in question to give a final +1

File: starboard/android/shared/test_filters.py [hunk 1]
```diff
@@ -39,12 +39,8 @@
         # the following tests depend upon.
         'VideoDecoderTests/VideoDecoderTest.ThreeMoreDecoders/*',
 
-        # The video pipeline will hang if it doesn't receive any input.
-        'PlayerComponentsTests/PlayerComponentsTest.EOSWithoutInput/*',
-
-        # The e/eac3 audio time reporting during pause will be revisitied.
-        'PlayerComponentsTests/PlayerComponentsTest.Pause/*ac3*',
-        'PlayerComponentsTests/PlayerComponentsTest.Pause/*ec3*',
+        # TODO(b/b/323250981): Flaky tests on Android
```
Comment (oxve) [inline starboard/android/shared/test_filters.py:42]: Added a TODO to the comment here

**Review by oxve — APPROVED** [review]

**Review by jellefoks — APPROVED** [review]

**Review by osagie98 — APPROVED** [review]

**Comment by zhongqiliang** [comment]
The change failed to submit because of these errors.
PS5 https://paste.googleplex.com/4802612245823488
TVOS https://paste.googleplex.com/4841300103266304

The code changed here should not be used by PS5 or TVOS, sync main branch to see if this error will go away.

**Comment by zhongqiliang** [comment]
Kokoro is still waiting for vote(s) for label(s) ["Linux ❔"].

Is it dead? How do I trigger it again?

**Comment by oxve** [comment]
> Kokoro is still waiting for vote(s) for label(s) ["Linux ❔"].
> 
> Is it dead? How do I trigger it again?

There's an unresolved comment on the change that is blocking merge. This check is not blocking.

It does look like it's misconfigured though, I can take a look today.

**Pull request merged**